src/app/_hooks/useIntersectionObserver.ts

```
@@ -49,7 +71,8 @@ export function useIntersectionObserver<T = string>(
     elements.forEach((element) => observer.observe(element));
 
     return () => {
-      elements.forEach((element) => observer.unobserve(element)); // observe the elements
+      elements.forEach((element) => observer.unobserve(element));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)